lib/streamlit/state/session_state.py

```diff
@@ -283,9 +283,6 @@ class SessionState(MutableMapping[str, Any]):
     """SessionState allows users to store values that persist between app
     reruns.
 
-    SessionState objects are created lazily when a script accesses
-    st.session_state.
-
```

Comment:
I removed this comment because it's not actually true - SessionState is always created for every script, currently.